dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -869,6 +909,44 @@ def main() -> None:
         )
         print(f"{tablename} metadata saved to {metadata_root_path}/")
 
+    print("Calculating failure notifications")
+    # Generate a map from SNS topic names -> list of test results that should
+    # be sent to that SNS topic
+    failures_by_topic: typing.Dict[str, typing.List[TestResult]] = {}
+    for test_category in test_categories:
+        for test_result in test_category.test_results:
+            if test_result.status == Status.FAIL and test_result.notify:
+                if test_result.notify not in failures_by_topic:
+                    failures_by_topic[test_result.notify] = []
+                failures_by_topic[test_result.notify].append(test_result)
+
+    # Generate the notification body and send it to each SNS topic. Start
+    # by parsing out a message body and subject for each group of failures
+    # by topic into a list of objects with the keys `topic_arn`, `subject`,
+    # and `body`
```

Comment:
**suggestion (non-blocking):** Let's add a call to action to the body as well. Something like "These were autogenerated by the Data Team. You will stop receiving these emails once the failures below are fixed" and then also our contact info at the bottom.

Reply:
I like that, done in c5096a8aeefab0bb059b8275aaf625a302fb69d0. I added our names instead of contact info, to avoid committing our email addresses to the source code; let me know if you don't mind the risk of spam and I can add email addresses in.